sql/schema.sql

```diff
@@ -466,15 +466,22 @@ CREATE TABLE public.hardware_product_profile (
     dimms_num integer NOT NULL,
     ram_total integer NOT NULL,
     nics_num integer NOT NULL,
-    sata_num integer,
-    sata_size integer,
-    sata_slots text,
-    sas_num integer,
-    sas_size integer,
-    sas_slots text,
-    ssd_num integer,
-    ssd_size integer,
-    ssd_slots text,
+    sata_hdd_num integer,
+    sata_hdd_size integer,
+    sata_hdd_slots text,
+    sas_hdd_num integer,
+    sas_hdd_size integer,
+    sas_hdd_slots text,
+    sata_ssd_num integer,
+    sata_ssd_size integer,
+    sata_ssd_slots text,
+    sas_ssd_num integer,
+    sas_ssd_size integer,
+    sas_ssd_slots text,
+    nvme_ssd_num integer,
```

Comment:
'nvme_ssd_size', 'nvme_ssd_slots', 'sas_ssd_size' and 'sas_ssd_slots' are not used anywhere. Do we need to add them?

Reply:
Before the current validation system, `*_size` and `*_slots` columns were actually validated on. When Lane switched us to the current validation system early last year, he did so with the intention of re-implementing validations for those types of things, but left before he was able to get around to doing so. It's my intention to (eventually) re-implement validations for those things.

Reply:
@daleghent sorry my friend! I remember trying to make the conversion to the new system exactly 1:1 with all the old validations. My apologies if I missed one and left more work for you to do.

Glad to see you extending the validation system in interesting ways!